src/psyclone/tests/psyir/nodes/omp_directives_test.py

```diff
@@ -113,6 +114,72 @@ def test_ompparallel_changes_begin_string(fortran_reader):
     assert pdir.children[2] is not priv_clause
 
 
+def test_ompparallel_structures_private_clause():
+    '''TODO'''
+    subroutine = Routine("testsub")
```

Comment:
While I'm impressed that you've constructed all the PSyIR for this by hand, I think your life would be much simpler (and the code shorter) if you 'cheated' and just used the `fortran_reader` test fixture with a suitable bit of Fortran.